FrontEndReact/src/View/Components/BackButton.js

```diff
@@ -0,0 +1,32 @@
+import Button from '@mui/material/Button';
```

Comment:
It seems like it would be best to me if BackButton and BackButtonResource were in a single file/class since they share a lot of code. Additionally, instead of hardcoding "Courses" as the new page, it would be nice if when the Back button component is created, a different page could be assigned so the same button could be used in many places.

Reply:
I have talked with Aldo and he will work on satisfying your request in the future Mr.Brennan!